Geometry/Fields/VectorFields/weightWindowsField_class.f90

```diff
@@ -0,0 +1,165 @@
+module weightWindowsField_class
+
+  use numPrecision
+  use genericProcedures, only : fatalError, numToChar
+  use dictionary_class,  only : dictionary
+  use dictParser_func,   only : fileToDict
+  use particle_class,    only : particle, particleState
+  use field_inter,       only : field
+  use vectorField_inter, only : vectorField
+
+  ! Tally Maps
+  use tallyMap_inter,             only : tallyMap
+  use tallyMapFactory_func,       only : new_tallyMap
+
+  implicit none
+  private
+
+  !!
+  !! Public Pointer Cast
+  !!
+  public :: weightWindowsField_TptrCast
+
+  !!
+  !! Weight Windows Vector Field
+  !!
+  !! Returns a vector with lower, upper and survival weight
+  !!
+  !! Sample Dictionary Input:
+  !!   varianceReduction { type weightWindowsField; file /home/WWfile.txt ;}
```

Comment:
Perhaps we should include some description of the expected contents of the file in the doc-comment?